### DISCUSSION TITLE
Trying to build Lazer

### DISCUSSION THREAD

**Discussion by LeafiDev** [ppy/osu#29018] [Q&A] [answered]
I've installed the .NET8.0 package from the README.md page. It does not restore workloads for some odd reason. What can I do about this?

**Comment by bdach** [comment]
post error messages

**Reply by LeafiDev** [reply]
unable to get them at the moment but dependencies are considered incompatible or missing
![image](https://github.com/user-attachments/assets/7d018cee-cad6-4775-8baf-3f079ea275aa)

**Reply by LeafiDev** [reply]
is there a specific version I'm looking for with these? I have .net8.0 SDK installed and running on the Desktop.slnf solution

**Reply by peppy** [reply]
You can't even read the error message in that view.

Please follow command line instructions from the readme and try again.

**Reply by LeafiDev** [reply]
quick question: Do I even need the repository for testing rulesets? My whole goal is to make a ruleset

**Reply by peppy** [reply, marked as answer]
![Safari 2024-07-23 at 07 21 40](https://github.com/user-attachments/assets/8efab2bb-2278-4d57-be69-4cc6950cc983)

**Reply by bdach** [reply]
in theory no, but if you can't get this repository to build, my confidence in you being able to create a ruleset project and get it to build is low

**Reply by LeafiDev** [reply]
Regardless I will work with what I have. Thank you.
